backend/src/main/java/com/google/backend/servlets/AlertsDataServlet.java

```diff
@@ -49,11 +54,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // TODO: only get alerts that the user is subscribed to.
     Query query = new Query(Alert.ALERT_ENTITY_KIND);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = datastore.prepare(query);
-    List<Alert> alertList = new ArrayList<>();
+    List<Entity> results = datastore.prepare(query).asList(FetchOptions.Builder.withLimit(MAX_LIMIT));    
```

Comment:
does the `MAX_LIMIT` mean you don't query all the alerts in the datastore if there is over `MAX_LIMIT` number of alerts?

Reply:
no, it's literally the max amount of objects you can query for from the Datastore

Reply:
can you set this to any number you want?

Reply:
yeah you could. I set it to the max limit so that I can get all the alerts but I'll be changing this per the timestamp sorting!